jest.config.js

```diff
@@ -1,7 +1,7 @@
 module.exports = {
   clearMocks: true,
   moduleFileExtensions: ['js', 'ts'],
-  roots: ['<rootDir>/packages'],
+  roots: ['<rootDir>/actions'],
```

Comment:
FYI I'm not a fan of a top-level jest config. Each package should be testable by itself. Can be used as "root config" tho, I guess.